src/Microsoft.VisualStudio.ProjectSystem.Managed.VS.UnitTests/ProjectSystem/VS/Tree/Dependencies/Snapshot/DependencyExtensionsTests.cs

```diff
@@ -50,31 +50,31 @@ public void ToViewModel()
                 KnownMonikers.AboutBox,
                 KnownMonikers.Abbreviation);
 
-            var dependencyResolved = IDependencyFactory.FromJson(@"
-{
-    ""ProviderType"": ""Yyy"",
-    ""Id"": ""tfm1\\yyy\\dependencyExisting"",
-    ""Name"":""dependencyExisting"",
-    ""Caption"":""DependencyExisting"",
-    ""HasUnresolvedDependency"":""false"",
```

Comment:
This change gets rid of `HasUnresolvedDependency`, which isn't even on `IDependency`. For this simple test type I prefer the strongly typed approach.